argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -4,41 +4,41 @@ metadata:
   name: build
 spec:
   source: # This will be replaced with a reference to your fork of this repo (see hack/patch-apps-for-dev.sh)
-    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
-    targetRevision: main
+    repoURL: https://github.com/jduimovich/infra-deployments.git
```

Comment:
missed reverting this, I think.

Reply:
My fault, I checked out the PR into my fork but pushed which updated the PR and not my branch from the fork

Reply:
re-ran upstream mode and pushed